aries_cloudagent/protocols/coordinate_mediation/v1_0/models/mediation_record.py

```diff
@@ -29,12 +29,16 @@ class Meta:
     STATE_REQUEST_RECEIVED = "request_received"
     STATE_GRANTED = "granted"
     STATE_DENIED = "denied"
-
+    
+    ORIGINATION_INTERNAL = "internal"
+    ORIGINATION_EXTERNAL = "external"
```

Comment:
I wonder if it might be clearer to express this as "role" rather than "origination" -- so client-server vs internal-external. Exactly what internal/external means isn't immediately obvious.

Reply:
I agree I will make the change.